services/secret-service/server/src/main/java/org/apache/airavata/mft/secret/server/backend/SecretBackend.java

```diff
@@ -55,4 +55,9 @@
     public DropboxSecret createDropboxSecret(DropboxSecretCreateRequest request) throws Exception;
     public boolean updateDropboxSecret(DropboxSecretUpdateRequest request) throws Exception;
     public boolean deleteDropboxSecret(DropboxSecretDeleteRequest request) throws Exception;
+
+    Optional<FTPSecret> getFTPSecret(FTPSecretGetRequest request) throws Exception;
```

Comment:
@gkiran292 @DImuthuUpe 
For keeping it uniform, I guess adding a public modifier to each of these declarations would be better.